src/MyTested.AspNetCore.Mvc.Authentication/ComponentBuilderAuthenticationWithoutExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace MyTested.AspNetCore.Mvc
+{
+    using System;
+    using System.Security.Claims;
+    using MyTested.AspNetCore.Mvc.Builders.Authentication;
+    using MyTested.AspNetCore.Mvc.Builders.Base;
+    using MyTested.AspNetCore.Mvc.Builders.Contracts.Authentication;
+    using MyTested.AspNetCore.Mvc.Builders.Contracts.Base;
+
+    /// <summary>
+    /// Contains authentication extension methods for <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/>.
+    /// </summary>
+    public static class ComponentBuilderAuthenticationWithoutExtensions
+    {
+        /// <summary>
+        /// Removes the <see cref="Claim"/> that have the provided claimType and value from <see cref="ClaimsPrincipal"/>
+        /// </summary>
+        /// <typeparam name="TBuilder">Class representing ASP.NET Core MVC test builder.</typeparam>
+        /// <param name="builder">Instance of <see cref="IBaseTestBuilderWithComponentBuilder{TBuilder}"/> type.</param>
+        /// <param name="claimType">The type of the claim that will be removed.</param>
+        /// <param name="value">The value of the claim that will be removed.</param>
+        /// <returns>The same component builder.</returns>
+        public static TBuilder WithoutUser<TBuilder>(
+            this IBaseTestBuilderWithComponentBuilder<TBuilder> builder,
+            string claimType, string value)
+            where TBuilder : IBaseTestBuilder
+            => builder.WithoutUser(user => user.WithoutClaim(claimType, value));
```

Comment:
The styling of these methods (which have multiple calls in a chain) should be:
`builder
    .WithoutUser(user => user
        .WithoutClaim(claimType, value));`

Update the whole file.